lib/monitor/measure.go

```diff
@@ -1,12 +1,12 @@
 /*
- * Time-measurement functions.
+ * measures functions.
  *
  * Usage:
- * ```measure := monitor.NewMeasure()```
- * ```// Do some calculations```
- * ```measure.MeasureWall("CPU on calculations")```
+ * ```measure := monitor.SingleMeasure("bandwidth")```
+ * or
+ * ```measure := monitor.NewTimeMeasure("round")
+ * ```measure.Record()```
  */
```

Comment:
Either put all usage in `monitor.go` before `package monitor`, or add this to the `type Measure interface` comment. Please also check if /`/`/` is correctly interpreted in godoc. I know that a 'tab' or an indention using spaces works as code-comment.